modules/ocf/manifests/motd.pp

```diff
@@ -1,12 +1,12 @@
-class ocf::motd {
-  $is_virtual = str2bool($::is_virtual)
-
-  if tagged('ocf_ssh') or tagged('ocf_apphost') {
+class ocf::motd($nfs_motd = false) {
```

Comment:
maybe we should have an attribute for whether a host is public, and then calculate this based on "public && nfs" (we already have an nfs fact)